ST_resgroups.c

```diff
@@ -713,64 +709,44 @@ void rgroup_IncrAges(void)
 			}
 		}
 	}
-
 }
 
-/***********************************************************/
-void RGroup_Update_Newsize(GrpIndex rg)
-{
-	/*======================================================*/
-	/* PURPOSE */
-	/*   Relative size for a group is 1.0 if all the group's
-	 *   species are established and size 1.0;
-	 */
+/************************************************************/
+/* Sums relsize for all individuals in all species in RGroup rg.
+   param rg = RGroup index.
+   Return: RGroup relsize. */
+RealF getRGroupRelsize(GrpIndex rg){
+    Int n;
+	SppIndex sp;
+	double sum = 0.0;
 
-	/* HISTORY */
-	/*   Chris Bennett @ LTER-CSU 5-Apr-2003            */
-	/*     Called from Species_Update_Newsize(), but safe to call
-	 *     from anywhere.
-	 *   7-Nov-03 (cwb) Added condition for annuals.  Annuals
-	 *     don't use the linked list of indiv objects because they
-	 *     come and go with each time step, so we just add their
-	 *     estimated size to the group size and move on.  See also
-	 *     Species_UpdateNewSize() and other calls with *annual*
-	 *     somewhere in them.
-	 */
+    ForEachEstSpp( sp, rg, n){
+		sum += getSpeciesRelsize(sp);
+    }
 
```

Comment:
@chaukap Note this is a fundamental change from the old implementation, as previously the species relsizes were summed for each rgroup, and then divided by the number of established species. In that sense it was an "average" relsize for the rgroup.

Old implementation:
`RGroup[rg]->relsize = sumsize / (RealF) RGroup[rg]->est_count;`

I am adding this comment because I don't think we have it documented anywhere. Of course with only 1 species turned on per rgroup, the output will be nearly identical between the old and new implementation. But once we turn on more than 1 species per rgroup, relsizes and biomass will be larger with our new implementation. 

I do believe this is the implementation we want - summing species relsizes - and I will let you know if I want to make any changes before we merge.